src/wms/src/main/java/org/geoserver/wms/featureinfo/FreeMarkerTemplateManager.java

```diff
@@ -146,8 +143,9 @@ private TemplateHashModel getStaticModel(String path)
                         return (TemplateHashModel) getStaticModels().get(path);
                     }
                 });
-        // as we want to look up different templates for each resource, the templates cannot
-        // be cached by name. Freemarker used to clear the cache when setting the loader,
+        // As we want to look up different templates for each resource, name cannot
+        // cache the templates.
```

Comment:
A name cannot cache, period :rofl: 

This change looks like a regression, caching happens "by name", as in, the name is the caching key.

Reply:
Done